docs/docsource/conf.py

```diff
@@ -54,6 +54,7 @@
             'sphinxcontrib.spelling',
             'sphinxcontrib.inkscapeconverter',
 	    'sphinx_version_ref',
```

Comment:
this line is out of alignment...

Reply:
hmm, funny, it wasn't on my editor. Probably a `tab` hidden :detective:

Reply:
done